src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Generated/Models/SasDefinitionBundle.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.KeyVault.Models
+{
+    using Azure;
+    using KeyVault;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// A sas definition bundle consists of key vault sas definition details
+    /// plus its attributes.
+    /// </summary>
+    public partial class SasDefinitionBundle
+    {
+        /// <summary>
+        /// Initializes a new instance of the SasDefinitionBundle class.
+        /// </summary>
+        public SasDefinitionBundle() { }
+
+        /// <summary>
+        /// Initializes a new instance of the SasDefinitionBundle class.
+        /// </summary>
+        /// <param name="id">The sas definition id.</param>
```

Comment:
We use "id" because all keys, secrets and certificates use it the same way, I think we should follow the pattern here, and if we really want to fix the naming/casing, we can do it for all together.